src/app/zap-templates/zcl/data-model/silabs/general.xml

```diff
@@ -170,32 +170,38 @@ limitations under the License.
-    <attribute side="server" code="0x000B" define="PRODUCT_URL" type="CHAR_STRING" length="64" writable="false" optional="true" introducedIn="l&amp;o-1.0-15-0014-04">product url</attribute>
-    <attribute side="server" code="0x0010" define="LOCATION_DESCRIPTION" type="CHAR_STRING" length="16" writable="true" default="" optional="true">location description</attribute>
-    <attribute side="server" code="0x0011" define="PHYSICAL_ENVIRONMENT" type="ENUM8" min="0x00" max="0xFF" writable="true" default="0x00" optional="true">physical environment</attribute>
-    <attribute side="server" code="0x0012" define="DEVICE_ENABLED" type="BOOLEAN" min="0x00" max="0x01" writable="true" default="0x01" optional="true">device enabled</attribute>
-    <attribute side="server" code="0x0013" define="ALARM_MASK" type="BITMAP8" min="0x00" max="0x03" writable="true" default="0x00" optional="true">alarm mask</attribute>
-    <attribute side="server" code="0x0014" define="DISABLE_LOCAL_CONFIG" type="BITMAP8" min="0x00" max="0x03" writable="true" default="0x00" optional="true">disable local config</attribute>
-    <command source="client" code="0x00" name="ResetToFactoryDefaults" optional="true" cli="zcl basic rtfd">
-      <description>Command that resets all attribute values to factory default.</description>
+
+    <attribute side="server" code="0x0000" define="INTERACTION_MODEL_VERSION" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">InteractionModelVersion</attribute>
+    <attribute side="server" code="0x0001" define="VENDOR_NAME" type="CHAR_STRING" length="32" writable="false" default="" optional="false">VendorName</attribute>
+    <attribute side="server" code="0x0002" define="VENDOR_ID" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">VendorID</attribute>
+    <attribute side="server" code="0x0003" define="MODEL_NAME" type="CHAR_STRING" length="32" writable="false" default="" optional="false">ModelName</attribute>
+    <attribute side="server" code="0x0004" define="PRODUCT_ID" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">ProductID</attribute>
+    <attribute side="server" code="0x0005" define="USER_LABEL" type="CHAR_STRING" length="32" writable="true" default="" optional="false">ProductID</attribute>
+    <attribute side="server" code="0x0006" define="LOCATION" type="CHAR_STRING" length="16" writable="true" default="" optional="false">Location</attribute>
+    <attribute side="server" code="0x0007" define="HARDWARE_REVISION" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">HardwareRevision</attribute>
+    <attribute side="server" code="0x0008" define="HARDWARE_REVISION_STRING" type="CHAR_STRING" length="64" writable="false" default="" optional="false">HardwareRevisionString</attribute>
+    <attribute side="server" code="0x0009" define="MANUFACTURING_DATE" type="CHAR_STRING" length="16" writable="false" default="" optional="true">ManufacturingDate</attribute>
```

Comment:
I guess you're out of luck. The spec attributes has changed since your patch. You will need to rebased, please re-request review once it is done.